EngineLayer/CrosslinkSearch/CrosslinkSpectralMatch.cs

```diff
@@ -49,14 +51,15 @@ public bool IsIntraCsm()
                 {
                     foreach (var beta in betaProteins)
                     {
-                        if (alpha == beta)
+                        if (alpha == beta || alpha == "DECOY_" + beta || beta == "DECOY_" + alpha)
```

Comment:
this counts decoy-target matches as intra? Add a comment since that's kinda important.